core/controllers/classifier.py

```diff
@@ -79,7 +79,10 @@ def verify_signature(message, vm_id, received_signature):
     if secret is None:
         return False
 
-    generated_signature = generate_signature(secret, message)
+    if message is None:
```

Comment:
Oh, just wanted to make sure that I start working on Oppia-ml part only when this thing is happening for sure (so like its final, right?).

I agree with your last comment, adding `message` in the key would be more transparent and hide the structure of message.